src/vcModals.cpp

```diff
@@ -656,10 +656,89 @@ void vcModals_DrawProjectInfo(vcState *pProgramState)
-        ImGui::TextWrapped("%s", pInfo);
+      {
+        if (udStrcmp(pProgramState->projectInfoTextures.pLastInfoText, pInfo) != 0)
+        {
+          pProgramState->projectInfoTextures.pLastInfoText = udStrdup(pInfo);
+          for (vcTexture *pTexture : pProgramState->projectInfoTextures.textures)
+            vcTexture_Destroy(&pTexture);
+          for (const char *pStr : pProgramState->projectInfoTextures.infoStrings)
+            udFree(pStr);
+          pProgramState->projectInfoTextures.infoStrings.Clear();
+          pProgramState->projectInfoTextures.textures.Clear();
+          pProgramState->projectInfoTextures.textureSizes.Clear();
+          
+          // Discover Images
+          int firstIndex = 0;
+          int startStrIndex = 0;
+          while (true)
+          {
+            size_t startImagePos = 0;
+            size_t openBracketPos = 0;
+            size_t closeBracketPos = 0;
+            size_t openParanthesisPos = 0;
+            size_t closeParanthesisPos = 0;
+
+            udStrchr(pInfo + firstIndex, "!", &startImagePos);
+            startImagePos += firstIndex;
+            udStrchr(pInfo + startImagePos, "[", &openBracketPos);
+            openBracketPos += startImagePos;
+            udStrchr(pInfo + openBracketPos, "]", &closeBracketPos);
+            closeBracketPos += openBracketPos;
+            udStrchr(pInfo + closeBracketPos, "(", &openParanthesisPos);
+            openParanthesisPos += closeBracketPos;
+            udStrchr(pInfo + openParanthesisPos, ")", &closeParanthesisPos);
+            closeParanthesisPos += openParanthesisPos;
+            
+            const int64_t invalidLen = udStrlen(pInfo);
+
+            if (startImagePos == invalidLen ||
+              openBracketPos == invalidLen ||
+              closeBracketPos == invalidLen ||
+              openParanthesisPos == invalidLen ||
+              closeParanthesisPos == invalidLen)
+              break;
+
+            size_t lastCharPos = closeParanthesisPos;
+            if (openBracketPos == startImagePos + 1 &&
+              closeBracketPos > openBracketPos &&
+              openParanthesisPos == closeBracketPos + 1 &&
+              closeParanthesisPos > openParanthesisPos)
+            {
+              const char *pSubStr = udStrndup(pInfo + startStrIndex, startImagePos - startStrIndex);
+              pProgramState->projectInfoTextures.infoStrings.PushBack(pSubStr);
+
+              const char *pTexPath = udStrndup(pInfo + (openParanthesisPos + 1), closeParanthesisPos - openParanthesisPos - 1);
+              uint32_t w, h;
+              vcTexture *pNewTexture = nullptr;
+              vcTexture_CreateFromFilename(&pNewTexture, pTexPath, &w, &h);
+              pProgramState->projectInfoTextures.textures.PushBack(pNewTexture);
+              pProgramState->projectInfoTextures.textureSizes.PushBack(ImVec2((float)w, (float)h));
+
+              firstIndex = (int)(lastCharPos + 1);
```

Comment:
This happens in both cases, and isn't modified - this should be set after `lastCharPos` is instead.

Reply:
Fixed